packages/integrations/markdoc/src/extensions/prism.ts

```diff
@@ -0,0 +1,22 @@
+// @ts-expect-error Cannot find module 'astro/runtime/server/index.js' or its corresponding type declarations.
+import { unescapeHTML } from 'astro/runtime/server/index.js';
```

Comment:
This type of comment broke the PR this morning. You should leave an empty newline after this import, so import sorting won't move the suppression comment. [Context](https://github.com/withastro/astro/pull/7206/files)

Reply:
Got it, was unaware of that! Will fix 👍